web/src/composebox_typeahead.ts

```diff
@@ -1095,21 +1095,6 @@ export function content_typeahead_selected(
         };
     }
 
-    // We only want to consider escape key for the stream+topic typeahead completion case.
-    if (event?.key === "Escape" && item.type !== "topic_list") {
-        setTimeout(() => {
-            // Select any placeholder text configured to be highlighted.
-            if (highlight.start && highlight.end) {
-                $textbox.range(highlight.start, highlight.end);
-            } else {
-                $textbox.caret(beginning.length);
-            }
-            // Also, trigger autosize to check if compose box needs to be resized.
-            compose_ui.autosize_textarea($textbox);
-        }, 0);
-        return beginning + rest;
-    }
```

Comment:
I don't quite understand how removing this is related to the new `escape_topic_completion` semantics?